docs/guide/dodona.md

```diff
@@ -22,9 +22,19 @@ In [Dodona](https://dodona.ugent.be), export the students' submissions:
 You can select multiple exercises in step 2 if the submissions are different enough that they will have less similarities. However, Dolos will still compare the submissions as if they were for a single exercise.
 :::
 
-## Run Dolos plagiarism detection
+## Run Dolos online
 
-You can run Dolos directly on the ZIP-archive:
+Go to [dolos.ugent.be/server](https://dolos.ugent.be/server) and upload the ZIP-archive you just downloaded from Dodona and click "Analyze".
+
+After a few seconds, the report will be ready and you will be able to explore the results.
+
+[Read more about how to use the Dolos Server](/guide/server).
```

Comment:
I think the naming here is a bit confusing. You call our hosted version "dolos server" but to me this suggests that I need to run my own server somehow. Below, you're talking about "dolos web" which is about the web interface of the self-hosted version but to me the name suggests it's a version I can access on the web.

Reply:
I agree `dolos-web` isn´t a good name. It is short for "Dolos web UI", but maybe "Dolos UI" would be better then...

Using "Dolos Online" is maybe a better alternative.